standard/statements.md

```diff
@@ -270,15 +270,18 @@ In addition to the reachability provided by normal flow of control, a labeled st
 
 ### 12.6.1 General
 
-A *declaration_statement* declares a local variable or constant. Declaration statements are permitted in blocks, but are not permitted as embedded statements.
+A *declaration_statement* declares a local variable, local constant, or local function. Declaration statements are permitted in blocks, but are not permitted as embedded statements.
```

Comment:
I can't remember offhand what an embedded statement is, which means another reader might not either - maybe link to it?

Reply:
Ah, it's saying that a declaration statement doesn't count as an embedded statement. Makes more sense.

Reply:
(Not a change for local functions, so resolving.)